Documentation/linux-install-diagnosis.md

```diff
@@ -0,0 +1,144 @@
+# Linux package manager install failure diagnosis
```

Comment:
I think it would be good to clarify the scope of the diagnosis to the packages we put up in the Microsoft feed as opposed to distro maintained feeds (eg, Red Hat).